### ISSUE TITLE
item_frequencies for 0 and False values

### ISSUE THREAD

**Issue by wpjunior** [hmarr/mongoengine#540]
test case.

``` python
from mongoengine import *
connect('teste')



class Teste(Document):
    val = IntField()
Teste.drop_collection()
t = Teste()
t.val = 0
t.save()

assert Teste.objects.item_frequencies('val') == {'0': 1}
```

**Comment by wpjunior** [comment]
at line: https://github.com/hmarr/mongoengine/blob/master/mongoengine/queryset.py#L1725

correct:
typeof field !== 'undefined'

**Comment by rozza** [comment]
Fixed in MongoEngine/mongoengine#40
